integration-tests/README.md

```diff
@@ -0,0 +1,19 @@
+
+## Steps to Run Locally
+1. Have docker installed
+2. Open a command prompt
+3. `cd <REPO_ROOT>/integration-tests/docker`
+4. `docker-compose up --force-recreate`
+5. `cd <REPO_ROOT>/integration-tests`
```

Comment:
The `docker-compose up` needs a `-d` (Arguably, this could also be done as-is in a second terminal session.)